CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 ## 5.20.0 (Unreleased)
 
+FEATURES:
+* **New Resource:** google_clouddeploy_custom_target_type_iam_* ([#17445](https://github.com/hashicorp/terraform-provider-google/pull/17445))
+
+IMPROVEMENTS:
+* bigquery: fixed `dataset_id` validation in `google_bigquery_dataset` ([#17449](https://github.com/hashicorp/terraform-provider-google/pull/17449))
```

Comment:
this looks like a bugfix? And it would be nice to say a little more about the user impact